.vscode/settings.json

```diff
@@ -9,6 +9,8 @@
   "rust-analyzer.check.allTargets": true,
   "rust-analyzer.check.features": "all",
   "rust-analyzer.server.path": "${workspaceFolder}/bin/rust-analyzer",
+  "rust-analyzer.check.command": "clippy",
```

Comment:
nit: suggest keeping entries alphabetically sorted.

Reply:
Done in f1c9bfb7f799adec4bd344436aac4779a4a71129